canvas_modules/harness/test_resources/parameterDefs/checkbox_paramDef.json

```diff
@@ -135,6 +135,9 @@
         "label": {
           "default": "checkbox default"
         },
+				"helper_text": {
```

Comment:
Looks like the formatting is off

Reply:
I have formatted this file using prettier is this fine or should I revert it and only format my changes ?

Reply:
You can format the entire file.  Just make sure it using the standard json formatting.

Reply:
Sure. I have updated accordingly.